pymilvus_orm/collection.py

```diff
@@ -561,8 +561,9 @@ def index(self, index_name="") -> Index:
         # TODO(yukun): Need field name, but provide index name, require some impl in server
         conn = self._get_connection()
         tmp_index = conn.describe_index(self._name, "")
+        field_name = tmp_index.pop("field_name")
```

Comment:
better to pass in a default value when calling pop to avoid throwing exceptions.